packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -66,9 +66,11 @@ export class VaButton {
   @Prop({ reflect: true }) secondary?: boolean = false;
 
   /**
-   * If `true`, the button will submit form data when clicked.
+   *  Having this attribut present will set the type of this button as 'submit'.
+   *  A value of: `prevent` --will  triger the onsubmit callback on form, but won't submit the form;
+   * `skip` --will submit the form but not trigger onsubmit callback;
    */
-  @Prop() submit?: boolean = false;
+  @Prop() submit?: string;
```

Comment:
Will this break any existing implementations?

Reply:
The reply left here is not publicly visible so here is the[ Slack convo](https://dsva.slack.com/archives/G01BJ3ESXL4/p1721234085222369?thread_ts=1721062755.570779&cid=G01BJ3ESXL4) and a screenshot of Ray's reply.

![Uploading Screenshot 2024-07-23 at 8.37.50 AM.png…]()